src/multisig/keypair.rs

```diff
@@ -5,7 +5,30 @@ use tofn::{
     sdk::api::{MessageDigest, SecretRecoveryKey},
 };
 
-pub type CurrentNetwork = snarkvm::prelude::TestnetV0;
+#[cfg(not(any(
+    feature = "aleo-testnet",
+    feature = "aleo-mainnet",
+    feature = "aleo-canary"
+)))]
+compile_error!(
+    "One of aleo networks should be enabled: 'aleo-testnet', 'aleo-mainnet', or 'aleo-canary'"
+);
+
+#[cfg(all(
+    feature = "aleo-testnet",
+    feature = "aleo-mainnet",
+    feature = "aleo-canary"
+))]
+compile_error!("Features 'aleo-testnet', 'aleo-mainnet' and 'aleo-canary' are mutually exclusive");
```

Comment:
This still allows for two features to be selected.

That would still result in a compile-time error because of how we use the features to define the `CurrentNetwork`, but the fix is rather straightforward:

```suggestion
#[cfg(any(
    all(feature = "aleo-testnet", feature = "aleo-mainnet"),
    all(feature = "aleo-testnet", feature = "aleo-canary"),
    all(feature = "aleo-mainnet", feature = "aleo-canary"),
))]
compile_error!(
    "Only one of 'aleo-testnet', 'aleo-mainnet', or 'aleo-canary' may be enabled at a time."
);
```

There are also other ways to address this:
- have a `const` assertion somewhere in the code
- look at the active feature flags from within build.rs

But I think the suggestion above is more closely related to your version.

Reply:
The alternative I mentioned:
```rust
const _: () = {
    let feature_count = 0
        + if cfg!(feature = "aleo-testnet") { 1 } else { 0 }
        + if cfg!(feature = "aleo-mainnet") { 1 } else { 0 }
        + if cfg!(feature = "aleo-canary") { 1 } else { 0 };

    assert!(
        feature_count == 1,
        "Exactly one of 'aleo-testnet', 'aleo-mainnet', or 'aleo-canary' must be enabled"
    );
};
```

That feels kinda janky to me, but it works.

Reply:
Thanks, good catch.
Handled at: 037716ae69f0debaf91e49610075dcd7b7227d28